isaric/parsers/isaric-ccpuk.toml

```diff
@@ -428,15 +428,3 @@
     { field = "pronevent_prtrt", ref = "Y/N/NK", description = "Prone ventilation" },
     { field = "daily_prone_cmtrt", ref = "Y/N/NK", description = "Prone positioning" },
   ]
-
-[visit.treatment_tocilizumab] # TODO: antiviral_cmtrt == 9 condition, but multiple fields, needs to be fixed
```

Comment:
Any reason these two were deleted?

Reply:
I am wondering the same ..

Reply:
Was giving validation errors, added back in with corrected field names.